Dockerfile.windows

```diff
@@ -0,0 +1,21 @@
+# escape=`
+
+ARG DOCUUM_VERSION=0.16.2
+
+FROM yodal/rust-windows AS build
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'));`
+    choco install -y docker-cli;
+
+ARG DOCUUM_VERSION
+RUN Invoke-WebRequest "https://github.com/stepchowfun/docuum/archive/v$($env:DOCUUM_VERSION).zip" -OutFile docuum.zip; `
+    Expand-Archive docuum.zip .; `
+    cd docuum-$env:DOCUUM_VERSION; `
+    $env:RUSTFLAGS = '-C target-feature=+crt-static'; `
+    cargo build --release"
+
+FROM mcr.microsoft.com/windows/nanoserver:1809
+ENTRYPOINT [ "docuum" ]
+COPY --from=build C:\ProgramData\chocolatey\lib\docker-cli\tools\docker.exe .
+ARG DOCUUM_VERSION
+COPY --from=build C:\docuum-${DOCUUM_VERSION}\target\release\docuum.exe .
```

Comment:
Since this file is called `Dockerfile.windows`, maybe we should rename the other Dockerfile to `Dockerfile.unix`?

Reply:
(Also can we add a line break to the end of this file for consistency with all the other files?)